src/components/LandingView/LandingView.js

```diff
@@ -0,0 +1,41 @@
+import React, { useState, useEffect } from 'react';
+import styles from './LandingView.module.css';
+import IntroductoryPage from './IntroductoryPage/IntroductoryPage';
+import AboutUs from './AboutUs/AboutUs';
+
+const LandingView = () => {
+  const [slide, setSlide] = useState(1);
+
+  useEffect(() => {
+  }, [slide]);
```

Comment:
- [ ] What is this doing?

Reply:
Hello @MuriloRoque, this is to connect the handling of the bullets and its slide card.